### DISCUSSION TITLE
Bring the old home page

### DISCUSSION THREAD

**Discussion by TheHellTower** [community/community#66167] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello,

I'm sorry but this update kinda sucks the fact we had the choice to see activities on OUR stuff and one for OTHER stuff stared/people followed, etc.. was really the best update I seen so far to this page..

You should bring it back.

P.S: You can see the following
- https://github.com/orgs/community/discussions/66166
- https://github.com/orgs/community/discussions/66165
- https://github.com/orgs/community/discussions/66158
- https://github.com/orgs/community/discussions/66164
- https://github.com/orgs/community/discussions/66164
And more..
Best Regards.

**Comment by VestOfHolding** [comment]
Agreed. The previous feed was fantastic for that choice, and on this new feed I actually can't even see the updates on my stuff anymore. So at minimum that's broken.
